packages/connector-alipay/README.md

```diff
@@ -1,2 +1,41 @@
-### Alipay Web Social Connector README
-placeholder
+# Alipay Web
+
+Alipay Web connector is designed for desktop Web applications. It takes advantage of Alipay's OAuth 2.0 authentication workflow and enables Alipay users to sign in to other Apps using public Alipay user profiles without going through a troublesome register process.
+
+## Register Alipay developer account
+
+[Register an Alipay developer account](https://certifyweb.alipay.com/certify/reg/guide#/) if you don't have one.
+
+## Create and configure Alipay app
+
+1. Sign in to the Alipay console with the account you have just registered.
```

Comment:
Better add a console link, it may be hard to find.

Reply:
Good, fixed.